server/athenian/api/openapi/openapi.yaml

```diff
@@ -2245,7 +2297,6 @@ components:
             Sets of developers and repositories for which to calculate the metrics.
             The aggregation is `AND` between repositories and developers.
             The aggregation is `OR` inside both repositories and developers.
-            Any contribution role is matched: authors, reviewers, commenters, committers, releasers.
```

Comment:
mmm... changing other things in this PR... ok...
Let me confirm:
- is this something new? or are you just updating the docs for a feature released some time ago?

Reply:
This doc is outdated. We switched to the fine-grained role filtering a month ago, or probably even older.